rqt_controller_manager/setup.py

```diff
@@ -0,0 +1,30 @@
+from glob import glob
+
+from setuptools import setup
+
+package_name = 'rqt_controller_manager'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages', ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        ("share/" + package_name + "/resource", glob("resource/*.*")),
+        ("share/" + package_name, ["plugin.xml"]),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='ijnek',
+    maintainer_email='kenjibrameld@gmail.com',
```

Comment:
Are you staying around to maintain the package? More than welcome to but I'd you'd rather not you can also change the contact info in here for mine.

Reply:
I put my name in to ensure I don't push work on to you, but I believe things are going to be easier if you are the maintainer. If you're OK with me putting your name down, I'd appreciate that.

Reply:
Yeah that's my job here ;)